.github/ISSUE_TEMPLATE/comp_resourcedetectors_aws.md

```diff
@@ -0,0 +1,41 @@
+---
+name: OpenTelemetry.ResourceDetectors.AWS
+about: Issue with OpenTelemetry.ResourceDetectors.AWS
+labels: comp:resourcedetectors.aws
+---
+
+# Issue with OpenTelemetry.Contrib.Extensions.AWSXRay
```

Comment:
Change the name here.

Reply:
da141f4736393382bbb32551f886d3d509f0c9d6